compiler/res/prog8lib/c64/textio.p8

```diff
@@ -484,10 +492,37 @@ asmsub  setchr  (ubyte col @X, ubyte row @Y, ubyte character @A) clobbers(A, Y)
 		bcc  +
 		inc  _mod+2
 +		pla
-_mod		sta  $ffff		; modified
+_mod		
+        sta  $ffff		; modified
+		rts
+
+_screenrows	.word  screen_addr + range(0, 1000, 40)
+_ptr    = setchr._mod +1
+	}}
+}
+
+asmsub  setcol  (ubyte col @X, ubyte row @Y, ubyte color @A) clobbers(A, Y)  {
```

Comment:
there's already setclr() that does the same thing, this new routine seem redundant?

Reply:
yes redundant